packages/expo-localization/CHANGELOG.md

```diff
@@ -4,13 +4,21 @@
 
 ### 🛠 Breaking changes
 
+- `Localization.region` changed from `undefined | string` to `null | string` on web to match iOS.
```

Comment:
Would you please change format of those entries to comply with what we used to do?

- Changes description ([#PRNUMBER](https://link.to/PR) by [@EvanBacon](https://link.to/your/profile))